install_config/configuring_sdn.adoc

```diff
@@ -160,6 +160,37 @@ but not to *172.28.0.0/14*, because even though the original range is entirely
 inside the new range, the original range must be at the start of the CIDR. See
 xref:expanding-the-service-network[Expanding the service network] for more
 information.
+
+
+.Changing the VXLAN PORT for the cluster network
+
+A cluster administrator can change the VXLAN port the system uses by changing the vxlanPort variable in the 
+master configuration file. 
+
+----
+networkConfig:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14 
+    hostSubnetLength: 9 
+  - cidr: 10.132.0.0/14
+    hostSubnetLength: 9
+  externalIPNetworkCIDRs: null
+  hostSubnetLength: 9
+  ingressIPNetworkCIDR: 172.29.0.0/16
+  networkPluginName: redhat/openshift-ovs-multitenant 
+  serviceNetworkCIDR: 172.30.0.0/16
+  vxlanPort: 4789 <1>
+----
+<1> vxlanPort is the value used by the nodes for the VXLAN Port. It can be valid port number, if unspecified 
```

Comment:
Better to give a valid port number range here.

Reply:
I think we can remove 'It can be valid port number' text, it seems obvious?